### PR TITLE
*: remove the remaining references of tikv-importer

### PR DESCRIPTION
**Pull request by kennytm**
## What have you changed? (mandatory)

Some non Rust files still references the tikv-importer binary, which may break the CI.

## What are the type of the changes? (mandatory)

The currently defined types are listed below, please pick one of the types for this PR by removing the others:

- Misc (other changes)

## How has this PR been tested? (mandatory)

N/A

## Does this PR affect documentation (docs) or release note? (mandatory)

No

## Does this PR affect tidb-ansible update? (mandatory)

No

## Refer to a related PR or issue link (optional)

#5036.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -38,7 +38,6 @@ COPY ./components/tipb_helper/Cargo.toml ./components/tipb_helper/Cargo.toml
 RUN mkdir -p ./src/bin && \
     echo 'fn main() {}' > ./src/bin/tikv-ctl.rs && \
     echo 'fn main() {}' > ./src/bin/tikv-server.rs && \
-    echo 'fn main() {}' > ./src/bin/tikv-importer.rs && \
```
Comment (overvenus) [inline Dockerfile]: Have you tested the changes locally?

Comment (kennytm) [reply Dockerfile]: This is the result of `gen-dockerfile.sh`...

Just tried `docker build .` and nothing special happened. What else to test?

Comment (siddontang) [reply Dockerfile]: if you can build docker successfully, it is fine.

**Pull request merged**